src/store/eventing/staged.ts

```diff
@@ -18,6 +18,10 @@ export const getStatus = (prevStaged: CreatorItem, nextStaged: CreatorItem): Sta
         status = 'update';
```

Comment:
Could be simplified a bit to `status = nextStaged === null ? 'cancel' : 'update';`